go.mod

```diff
@@ -17,12 +17,12 @@ require (
 	github.com/pborman/uuid v1.2.1
 	github.com/pkg/errors v0.9.1
 	github.com/prometheus/client_golang v1.16.0
-	github.com/rancher/apiserver v0.0.0-20230831052300-120e615b17ba
-	github.com/rancher/dynamiclistener v0.3.6-rc2.0.20230831052350-0132d96ec2c5
+	github.com/rancher/apiserver v0.0.0-20240124133525-3e714782f883
+	github.com/rancher/dynamiclistener v0.4.0-rc1
```

Comment:
This bump includes my wrangler v2 commit as well as https://github.com/rancher/dynamiclistener/commit/0a2d8dff62fde6dfaf009141c12b30c316d8cff3

I believe during the last rancher release we forgot to un-RC the Steve deps to use [dynamiclistener@v0.3.6](https://github.com/rancher/dynamiclistener/releases/tag/v0.3.6)
Since that is what the late Rancher release of [v2.8.1 ](https://github.com/rancher/rancher/releases/tag/v2.8.1)uses https://github.com/rancher/rancher/blob/v2.8.1/go.mod#L114

commit dif: https://github.com/rancher/dynamiclistener/compare/0132d96ec2c5...v0.4.0-rc1